lib/src/source_visitor.dart

```diff
@@ -716,37 +716,50 @@ class SourceVisitor extends ThrowingAstVisitor {
       //       parameter,
       //     ) : super();
       space();
+
+      var isOptional =
+          node.parameters.parameters.last.kind != ParameterKind.REQUIRED;
+      if (node.initializers.length > 1) {
+        _writeText(isOptional ? " " : "  ", node.separator.offset);
```

Comment:
[optional] It's hard for me to tell the difference between `" "` and `"  "` as I'm scanning the code.

Maybe make this `" "` and `" " * 2`?

Reply:
Hmm, that feels a little gratuitous.